projects/wallet-framework-dotnet.md

```diff
@@ -0,0 +1,65 @@
+# Project Name
+Wallet Framework .NET (wallet-framework-dotnet)
+
+# Project Description
+The wallet-framework-dotnet is a framework designed for .NET, focusing on providing a multi-platform wallet framework. Initially a part of the Hyperledger Aries project (Aries Framework .NET), this initiative has now branched out to cater to a broader audience. The primary aim is to create a multiprotocol wallet framework enabling implementations of OpenID4VC and SD-JWT VC, in accordance to the European Identity Wallet initiative's objectives.
+
+Currently, the framework supports DidComm v1 and AnonCreds. There is an active intention to extend support for other promising protocols, notably DidComm v2, to ensure the framework remains at the forefront of digital identity solutions.
+
+Furthermore, the team is considering the development of SD-JWT credentials as a standalone library. This might transition into a separate project proposal in the future, underscoring the commitment to modular and reusable components in the digital identity space.
+
+# Alignment with the OpenWallet Foundation Mission
+The wallet-framework-dotnet directly aligns with the OpenWallet Foundation mission by offering an open-source solution that will advance the state of digital identity wallets. The project's focus on implementing OpenID4VC in conjunction with the European Identity Wallet initiative underscores our commitment to global, interoperable standards, and enhances trust and security in digital identity systems.
+
+# Current Code of Conduct
+tbd
+
+# TAC Sponsor
+To be identified.
+
+# Project License
+Apache 2.0
+
+# Source Control
+To be established (Expected: https://github.com/openwallet-foundation/wallet-framework-dotnet)
+
+Suceeds Hyperledger Aries project: [aries-framework-dotnet](https://github.com/hyperledger/aries-framework-dotnet)
+
+# Issue Tracker
+GitHub Issues
+
+# External Dependencies
+- AnonCreds-RS
+- Indy-VDR
+- Aries-Askar
+
+# Release Methodology
+The project will be distributed as nuget package.
+
+# Initial Maintainers
+- Sebastian Bickerle [ntsbs](https://github.com/ntsbs)
+- Christopher Hempel [CHempel-esatus](https://github.com/CHempel-esatus)
+
+# Proposed Project Governance
+tbd
+
+# Links to Documented Governance Practices
+tbd
+
+# Preferred Maturity Level
+Labs
+
+# Communication Channels
+To be established. (Discord Channel in OWF)
+
+# Project Website
+GitHub project
+
+# Social Media
+
+
+# Financial Sponsorship
+
+
+# Infrastructure
+- Requires build and release pipeline for the Nuget package
```

Comment:
Can this be handled by GitHub Actions?

Reply:
I personally have experience with Azure Devops, but I am certain that GitHub Actions would also work

Reply:
@ntsbs please don't use ADO - Microsoft's stated plan is to EOL ADO once GHA has all the features of ADO. It is also harder to manage.

Reply:
I am okay with Github Actions

Reply:
@ntsbs Will this be a possible solution?

> You can configure the dotnet command-line interface (CLI) to publish NuGet packages to GitHub Packages and to use packages stored on GitHub Packages as dependencies in a .NET project.

https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry

Reply:
If this can be set up with OWF, I am happy to do so